--- disk_usage_original.py	2021-01-06 16:33:52.295000000 -0500
+++ disk_usage_fixed.py	2021-01-07 10:56:26.624970526 -0500
@@ -8,6 +8,7 @@
 """
 
 import shutil
+import sys
 
 
 def check_disk_usage(disk, min_absolute, min_percent):
@@ -23,10 +24,9 @@
 
 
 # Check for at least 2 GB and 10% free
-if not check_disk_usage("/", 2*2**30, 10):
+if not check_disk_usage("/", 2, 10):
     print("ERROR : Not enough disk space")
-    return 1
-
+    sys.exit(1)
 
 print("Everything OKAY")
-return 0
+sys.exit(0)
